pyserini/encode/_qry_encoders.py

```diff
@@ -0,0 +1,224 @@
+#
```

Comment:
Why aren't doc and query encoders grouped together?

Why not an organization where each family is put in a separate file?

E.g., `ance.py`, `dpr.py`, etc.?

Reply:
yeah it make sense